### PR TITLE
[MIG] website_sale_attribute_filter_form_submit: Migration to 13.0

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa TT30550

please review @victoralmau @ernestotejeda 

ping @miguels73

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]
Code review OK.
Please, squash administrative commits to reduce commits.

File: website_sale_attribute_filter_form_submit/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+To configure this module, you need to:
+
+#. Go to the frontend /shop url of the website you want to set the manual filters into.
+#. Hit the Configure menu and enable the option *Manual filters*.
```
Comment (ernestotejeda) [inline website_sale_attribute_filter_form_submit/readme/CONFIGURE.rst:5]:
```suggestion
#. Hit the Configure menu and enable the option *Product Attribute's Filters*.
#. Hit the Configure menu and enable the option *Manual filters*.
```

File: website_sale_attribute_filter_form_submit/static/src/js/website_sale.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Tecnativa - David Vidal
+// License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+odoo.define("website_sale_attribute_filter_form_submit.website_sale", function(
+    require
+) {
+    "use strict";
+
+    var sAnimations = require("website.content.snippets.animation");
+
+    sAnimations.registry.WebsiteSale.include({
```
Comment (ernestotejeda) [inline website_sale_attribute_filter_form_submit/static/src/js/website_sale.js]: Should we use publicWidget here instead? ping @Tardo 
```
publicWidget.registry.WebsiteSale.include({
    ...................
});
```

**Review by ernestotejeda — CHANGES_REQUESTED** [review]
Functional tests OK.

**Comment by chienandalu** [comment]
Thanks @ernestotejeda changes done

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**